app/assets/stylesheets/scaffold.css

```diff
@@ -0,0 +1,80 @@
+body {
+  background-color: #fff;
+  color: #333;
+  margin: 33px;
+}
+
+body, p, ol, ul, td {
+  font-family: verdana, arial, helvetica, sans-serif;
+  font-size: 13px;
```

Comment:
You should define your font size at ```html``` level.

Reply:
it is a default file generated by rails. That should be removed